src/Github.js

```diff
@@ -267,16 +277,20 @@ class Github {
             }`
         let issues = [];
         // await Promise.all([]);
-        for (const repo of searchRepos) {
+        let resultSets = searchRepos.map(repo => {
             let [owner, project] = repo.split('/');
-            let results = await graphql(query, {
+            return graphql(query, {
                 headers: {
                     authorization: `token ${token}`
                 },
                 owner: owner,
                 project: project,
                 labels: labels
             });
+        });
+        Promise.all(resultSets);
```

Comment:
You only need `Promise.all` if you want to `await` the entire array of promises. The actual work has already started (back when `graphql` was called). It should be safe to remove this, as other bits are ensuring all requests are issued in parallel.